src/translator/aligned.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#include <cstdlib>
+#include <new>
+#ifdef _MSC_VER
+#include <malloc.h>
+#endif
+
+// Aligned simple vector.
+
+namespace intgemm {
```

Comment:
Different namespace please, otherwise we get the same object defined twice in the binary.